packages/elements/src/storybook/concepts.stories.mdx

```diff
@@ -0,0 +1,92 @@
+
+We also ship a number of helper clases to handle things like `flexbox` sizing and `spacing` with margins and padding. See the other stories in this guide for examples.
+
+For non-React users, just add the classes to your markup you see in the rendered markup `canvases` on each page of the docs.
+
+## Inheritance of HTML properties
+
+All of our components inherit via props their underlying HTML properties - we apply these using the `{...rest}` operator and for `TypeScript` users this behaviour is strongly typed.
+
+For example, if I were to import an `Input` from elements, I can apply via props, all of the native props of a react `input`, contrived example below;
+
+```tsx
+import { Input } from "@reapit/elements"
+
+
+const MyComponent = () => (
+  <Input type="text" onChange={(e) => console.log('Pass me to your form library!', e)} />
+)
+```
+
+This is a very importand concept when working with Elements as it means you can integrate our components with any additional libraries or custom behaviours you wish, using the native React API. 
+
+In addtion all of our non-closing tags render React `children` so that you can wrap any content you choose within them.
+
+## Intents
+
+Intent is another concept we use widely in the library. 
+
+For elements like buttons, icons and notifications, it is common for them to come in "flavours" like "warning" or "success". We have normalised this accross our components so where relevant, you can apply an `intent` prop and it will style the component appropriate to that intent eg;
+
+```tsx
+import { Icon } from "@reapit/elements"
+
+// Will render a "danger" icon in red
+const MyComponent = () => (
+  <Icon intent="danger" icon="systemAdd" />
+)
+```
+
+The various intent types are below and are consistent accross the library.
+
+| Intent    | Usage                                                                                                  |
+| --------- | ------------------------------------------------------------------------------------------------------ |
+| primary   | The main/most likely action a user might want to take                                                  |
+| secondary | Typically used alonside a primary button to offer an alternative action, e.g. a Cancel or Close button |
+| critical  | An action that should be the very next, or only logical action that a user can take                    |
+| success   | An action that would lead to a good or pleasant outcome                                                |
+| danger    | An action that would lead to a potentially bad or dangerous outcome, e.g a delete button               |
+| neutral   | An action that has no specific meaning - essentially no intent is inferred from                        |
```

Comment:
Stupid suggestion. Could we add coloured emoji's for the intents? 
https://stackoverflow.com/questions/11509830/how-to-add-color-to-githubs-readme-md-file/65174367#65174367

Reply:
Erm, maybe... TBH, not sure how they would look in Storybook though...

Reply:
sure, was just an idea :)